spec/models/media_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "spec_helper"
+
+describe Media do
+  describe "#translation_key" do
+    it "returns a parameterized version of the media name using underscore as a separator" do
+      media = FactoryBot.create(:media, name: "Widgets; Tools & Doodads")
```

Comment:
Will `create(:media, name: etc)` work without the `FactoryBot.` here?

Reply:
It did!